rust/lance/examples/ivf_hnsw.rs

```diff
@@ -96,8 +97,11 @@ async fn main() {
 
     println!("Loaded {} records", dataset.count_rows(None).await.unwrap());
 
+    let take_projection =
+        ProjectionRequest::from_schema(dataset.schema().project(&[column]).unwrap());
+
     let q = dataset
-        .take(&[0], &dataset.schema().project(&[column]).unwrap())
+        .take(&[0], take_projection)
```

Comment:
I guess this could also be written a bit nicer as

```rust
ProjectionRequest::from_columns(&[column], dataset.schema())
```

Reply:
Done.